GameData/RealismOverhaul/RO_SuggestedMods/SCANSat/RO_SCANsat.cfg

```diff
@@ -53,6 +72,327 @@
 	
 	@MODULE[SCANsat]
 	{
-		@power = 0.2
+		@RESOURCE[ElectricCharge]
+		{
+			@rate = 0.2
+		}
+	}
+}
+//	=================================================================================
+//	NEW PARTS
+//	=================================================================================
+//	sensorType keys: add to add sensors
+//	2^0: Lo-res radar
+//	2^1: Hi-res radar
+//	2^2: Lo-res Visual
+//	2^3: Biome
+//	2^4: Anomaly
+//	2^5: Anomaly Detail
+//	2^6: Hi-res Visual
+//	2^7: Lo-res Resource
+//	2^8: Hi-res resource
+//	=================================================================================
+
+//	Lo-res radar
+//	=================================================================================
+
+// Lo-res altimetry (regular radar), based on Ranger Block II radar
+// Source: https://books.google.com/books?id=51QCAAAAIAAJ&pg=PA1004&lpg=PA1004&dq=ranger+5+radar+altimeter&source=bl&ots=QSh4sZdgyJ&sig=ACfU3U2E6TgvRqYEcohEXOlM6CImanplMg&hl=en&sa=X&ved=2ahUKEwjf76q16onqAhWUHc0KHdQLAfsQ6AEwBXoECA8QAQ#v=onepage&q=ranger%205%20radar%20altimeter&f=false
+@PART[scansat-radar-poseidon-3b-1]:LAST[SCANsat]
```

Comment:
Does SCANSat do a lot of its own patching of its modules in :FOR[SCANsat] that we need/want to run in :LAST instead of just in a typical :FOR[RealismOverhaul] pass?

Reply:
RSS has a patch the forces all parts with a scansat module to some incredibly unhelpful defaults, and it runs AFTER[SCANsat]. Since RSS patches run after RO patches, this means only LAST and FINAL will make any difference. I have also opened a PR that moves the RSS patches to a reasonable time (FOR), but in case this PR gets merged first I kept it on LAST

Reply:
I merged the RSS PR but we might still need to use the :Last pass since it isn't certain when and for which versions the next RSS release will be.

Reply:
Yep, my thoughts exactly